src/components/MakeWidget/subcomponents/RateField/RateField.styles.tsx

```diff
@@ -44,38 +29,26 @@ export const Wrapper = styled.div<{ isButton: boolean }>`
   text-transform: uppercase;
   font-size: 0.75rem;
 
-  ${({ isButton }) =>
-    isButton &&
-    css`
-      ${BorderedPill};
-      ${ButtonBorder};
-
-      color: ${({ theme }) =>
-        theme.name === "dark" ? theme.colors.white : theme.colors.primary};
-      gap: 0.125rem;
-
-      /* fixes pixel differences with bordered pill */
-
-      height: 2.5rem;
-      padding: 0 1rem;
-
-      cursor: pointer;
-    `}
-
   ${({ isButton, theme }) =>
-    !isButton &&
-    css`
+    isButton
+      ? LargePillButtonStyle
+      : `
       color: ${theme.colors.lightGrey};
       gap: 0.375rem;
+      
+      ${StyledIconButton} {
+        margin-left: -0.25rem;
+      }
 
       ${RateBox} {
         border-radius: 0.125rem;
         border: 1px solid ${theme.colors.borderGrey};
+        margin: 0;
+        height: 2rem;
+        padding: 0 0.5rem;
         font-family: ${fontMono};
         font-size: 0.875rem;
         font-weight: 500;
```

Comment:
```suggestion
        font-weight: 500;
        line-height: 1;
        text-align: center;
```

Reply:
Check! `text-align` is not needed but added the other suggestions.